content/guides/development.md

```diff
@@ -240,7 +240,301 @@ T> Now that your server is working, you might want to give [Hot Module Replaceme
+  |- package.json
+  |- webpack.config.js
+  |- /dist
+  |- /src
+    |- index.js
+    |- print.js
++   |- server.js
+  |- /node_modules
+```
+
+__src/server.js__
+
+``` js
+const express = require('express');
+const webpack = require('webpack');
+const webpackDevMiddleware = require('webpack-dev-middleware');
+
+const app = express();
+const config = require('../webpack.config.js');
+const compiler = webpack(config);
+
+// Tell express to use the webpack-dev-middleware and use the webpack.config.js
+// configuration file as a base.
+app.use(webpackDevMiddleware(compiler, {
+  publicPath: config.output.publicPath
+}));
+
+app.listen(3000, function () {
+  console.log('Example app listening on port 3000!');
+});
+```
+
+Now add an npm script to make it a little easier to run the server:
+
+__package.json__
+
+``` json
+  {
+    "name": "development",
+    "version": "1.0.0",
+    "description": "",
+    "main": "webpack.config.js",
+    "scripts": {
+      "test": "echo \"Error: no test specified\" && exit 1",
+      "watch": "webpack --progress --watch",
+      "start": "webpack-dev-server --open",
++     "server": "node src/server.js",
+      "build": "webpack"
+    },
+    "keywords": [],
+    "author": "",
+    "license": "ISC",
+    "devDependencies": {
+      "css-loader": "^0.28.4",
+      "csv-loader": "^2.1.1",
+      "express": "^4.15.3",
+      "file-loader": "^0.11.2",
+      "html-webpack-plugin": "^2.29.0",
+      "style-loader": "^0.18.2",
+      "webpack": "^3.0.0",
+      "webpack-dev-middleware": "^1.12.0",
+      "xml-loader": "^1.2.1"
+    }
+  }
+```
+
+Now in your terminal run `run npm server`, it should give you an output similar to this:
+
+``` bash
+Example app listening on port 3000!
+webpack built 27b137af6d9d8668c373 in 1198ms
+Hash: 27b137af6d9d8668c373
+Version: webpack 3.0.0
+Time: 1198ms
+          Asset       Size  Chunks                    Chunk Names
+  app.bundle.js    1.44 MB    0, 1  [emitted]  [big]  app
+print.bundle.js    6.57 kB       1  [emitted]         print
+     index.html  306 bytes          [emitted]
+   [0] ./src/print.js 116 bytes {0} {1} [built]
+   [1] ./src/index.js 403 bytes {0} [built]
+   [2] ./node_modules/lodash/lodash.js 540 kB {0} [built]
+   [3] (webpack)/buildin/global.js 509 bytes {0} [built]
+   [4] (webpack)/buildin/module.js 517 bytes {0} [built]
+Child html-webpack-plugin for "index.html":
+         Asset    Size  Chunks  Chunk Names
+    index.html  544 kB       0
+       [0] ./node_modules/html-webpack-plugin/lib/loader.js!./node_modules/html-webpack-plugin/default_index.ejs 538 bytes {0} [built]
+       [1] ./node_modules/lodash/lodash.js 540 kB {0} [built]
+       [2] (webpack)/buildin/global.js 509 bytes {0} [built]
+       [3] (webpack)/buildin/module.js 517 bytes {0} [built]
+webpack: Compiled successfully.
+```
+
+Now fire up your browser and go to `http://localhost:3000/app`, you should see your webpack app running and functioning!
+
+
+#### Adding Hot Module Replacement
```

Comment:
I would drop this whole section -- if anything just make a note of the `webpack-hot-middleware` package at the end of the section above and link to it. Or, even better, just add a small tip to the __Hot Module Replacement__ guide that notes and links to it, maybe something like:

> T> If you took the route of using `webpack-dev-middleware` instead of `webpack-dev-server`, please use the [`webpack-hot-middleware`](https://github.com/glenjamin/webpack-hot-middleware) package to enable HMR on your custom server or application.